en/docs/develop/manage-repository/connect-your-own-github-repository-to-choreo.md

```diff
@@ -1,148 +1,69 @@
-
-      ```
-       Compiling source
-         johnsmith/foo:0.1.0
-    
-       Generating executable
-         .choreo/build/bin/foo.jar
-       [main ea3deab] test
-       3 files changed, 7 insertions(+), 3 deletions(-)
-      ```
-12. Navigate to the **Choreo Console** tab to develop the component depending on your requirement.
-
-## Step 3: Design the REST API
-
-Follow the step-by-step instructions under [step 1.3](https://wso2.com/choreo/docs/get-started/tutorials/create-your-first-rest-api/#step-13-design-the-rest-api) in the Create Your First REST API tutorial. 
-
-## Step 4: Run and try out the REST API
-
-Follow the step-by-step instructions under [step 1.4](https://wso2.com/choreo/docs/get-started/tutorials/create-your-first-rest-api/#step-14-run-and-test-the-rest-api) in the Create Your First REST API tutorial.
-
-## Step 5: Commit the source code of the REST API component to your GitHub repository
-
-The REST API component you designed is currently available only in the Web Editor. To allow multiple developers to develop it in collaboration, you must commit the source code of the REST API component to the GitHub repository you connected.
-Follow these steps:
-
-1. Click **Sync with Choreo Upstream** in the bottom pane of the page.
-   ![Sync with upstream](../../assets/img/tutorials/connect-own-repo/sync-with-choreo-upstream.png){.cInlineImage-full}
-2. In the displayed message, click **Sync my changes with Choreo**.
-3. In the left pane, enter a commit message (for example, `Implement REST API`) and click the tick to commit.
-   ![Enter commit message](../../assets/img/tutorials/connect-own-repo/enter-commit-message.png){.cInlineImage-full}
-
-    This displays a message requesting you to confirm that you want to stage all your changes and commit them directly.
-
-4. Click **Yes**.
-
-    !!! info
-        If you are connecting your own repository to Choreo for the first time, you will see a message requesting you to configure your user name and email address in GitHub. Click Open Git Log, go to the **Terminal** tab, and run the following commands to configure your username and email address for GitHub commits:
-
-         `git config --global user.email "<email@example.com>"`
-
-         `git config --global user.name "<Your User Name>"` 
-
-5. To push changes to your GitHub repository, click **0↓ 1↑** in the bottom pane.
-   ![Push changes to repository](../../assets/img/tutorials/connect-own-repo/push-changes-to-repository.png){.cInlineImage-full}
+Developers can collaborate via any of the following approaches:
 
-    !!! info 
-        This icon is displayed only when you complete the commit process.
+- Use the same upstream repository to create a shared component in the same organization, develop, push changes to the respective tracking branch, and eventually send a PR to the relevant branch.
```

Comment:
Let's use thev term "pull request" instead of the abbreviation "PR".

Reply:
done